lib/twine/cli.rb

```diff
@@ -306,8 +306,13 @@ def self.prepare_description!(options, summary_width)
         end
       end
 
+      if IO.console
+        console_width = IO.console.winsize[1] if IO.console
```

Comment:
Do you need to check `if IO.console` again here if you already checked it above?